docs/src/main/sphinx/admin/resource-groups.md

```diff
@@ -187,6 +187,8 @@ documentation](https://docs.oracle.com/en/java/javase/21/docs/api/java.base/java
 
 - `group` (required): the group these queries will run in.
 
+All rules within a single selector are applied following an AND logic
```

Comment:
```suggestion
All rules within a single selector are combined using a logic `AND`, and therefore all rules must match for a selector to be applied.
```

Reply:
`logic AND` --> `logical AND` I assume?

Reply:
well.. either is fine grammatically from what I know .. maybe we can refer to the expert - @colebow  ?